### PR TITLE
README: links for advanced configurations to venn and hydra wiki entries

### PR DESCRIPTION
**Pull request by matu3ba**
Also mention core problem of mini.move to set expectations of users correctly.

### CODE REVIEW

**Comment by matu3ba** [comment]
I still need to create screenshots for the hydra.nvim wiki, but wanted to show my ideas. I hope this is sufficiently minimal redundancy.

File: README.md [hunk 1]
```diff
@@ -25,17 +25,38 @@ Usage
...
+[in the hydra wiki](https://github.com/nvimtools/hydra.nvim/wiki/Draw-diagrams)
+and the more extensive version including explanations
+[in this repo wiki](https://github.com/jbyuki/venn.nvim/wiki/Draw-diagrams).
+
+#### Moving selection
+
+Drawing and text repositioning is a frequent operation on more complex
+graphics. While `gR` allows to overwrite text from left to right, it does not
+allow to move (text) selections around.
+Using mini.move works with for now main usability restriction of
```
Comment (jbyuki) [inline README.md:53]: Add a link to mini.move ?

**Comment by jbyuki** [comment]
This is a welcome more in-depth explanation on how to use this kind of diagramming system. Feel free to merge when you think it's ready.

**Comment by jbyuki** [comment]
Awesome! 
I just made some small modifications in the wiki, mostly formatting.

**Pull request merged**